ui/src/components/Sidebar.tsx

```diff
@@ -31,6 +31,7 @@ export default function Sidebar(props: Props) {
                                 <AnchorLink href={'#' + lrdDocsItem.http_method + lrdDocsItem.uri}
                                     onClick={() => {
                                         window.history.pushState({}, '', '#' + lrdDocsItem.http_method + lrdDocsItem.uri);
+                                        setTimeout(() => {window.scrollBy(0, -128)}, 600); 
```

Comment:
This doesn't work 100% correctly

Reply:
Feel free to get rid of `AnchorLink` and use something else in order to get the scroll work accurately.